docs/docker.md

```diff
@@ -2,55 +2,103 @@
 
 # Docker
 
-This page list various operations that can be automated _via_ Docker.
+Production and development images for [Cozy Stack](https://cozy.io), Dockerfiles, docker-compose.yml & co lives [there](https://github.com/cozy/cozy-stack/tree/master/scripts)
 
-## Running a CouchDB instance
```

Comment:
Please don't remove the section for running a CouchDB instance via docker. It is useful when running the stack without docker (for coding the stack for example).

Reply:
It's still here, but included in the "The secure do it yourself way" code block. I'll reformat it with subtitle so it's more obvious. But I still prefer not having it as a top title, the doc being around running Cozy Stack and the steps required

Reply:
Je suis d'accord que cette doc est d'abord destinée aux contributeurs donc laisser quelque part les commandes docker pour couchdb isolément n'est pas inutile, même si ce n'est que décrit comme une alternative manuelle